applications/nrf5340_audio/src/bluetooth/bt_stream/unicast/unicast_server.c

```diff
@@ -73,22 +73,26 @@ static const uint8_t cap_adv_data[] = {
 	BT_AUDIO_UNICAST_ANNOUNCEMENT_TARGETED,
 };
 
-#define AVAILABLE_SOURCE_CONTEXT                                                                   \
-	(BT_AUDIO_CONTEXT_TYPE_UNSPECIFIED | BT_AUDIO_CONTEXT_TYPE_CONVERSATIONAL)
-
+#if defined(CONFIG_BT_AUDIO_RX)
 #define AVAILABLE_SINK_CONTEXT                                                                     \
 	(BT_AUDIO_CONTEXT_TYPE_UNSPECIFIED | BT_AUDIO_CONTEXT_TYPE_MEDIA |                         \
 	 BT_AUDIO_CONTEXT_TYPE_CONVERSATIONAL)
+#else
```

Comment:
Looks great. Please have a look at the sniffer to confirm.

Reply:
Done